docs/advanced-topics/hybrid-models.md

```diff
@@ -0,0 +1,137 @@
+# Hybrid model deployment
+
+FHE enables cloud applications to process private user data without running the risk of data leaks. Furthermore, deploying ML models in the cloud is advantageous as it eases model updates, allows to scale to large numbers of users by using large amounts of compute power, and protects model IP by keeping the model on a trusted server instead of the client device.
+
+However, not all applications can be easily converted to FHE computation and the computation cost of FHE may make a full conversion exceed latency requirements.
+
```

Comment:
I find the sentence a bit hard to follow. I propose


```suggestion
For certain applications, transitioning to FHE computation may not be straightforward due to inherent complexities. Additionally, the computational overhead of FHE might cause latency issues that surpass acceptable thresholds.
```

Reply:
agree !